packages/core/styles/v2/components/modal.scss

```diff
@@ -67,5 +67,6 @@
 
   .cove-icon {
     flex: 0 0 auto;
+    display: block;
```

Comment:
fix: add to the specific selector. This is specific to modals. If you can't find where to put it, place it somewhere inside the editor panel thats specific to the accordion you're working on.

Reply:
Removed.  This was added as a test to see if it had any effect.  It did not but I admit I forgot to remove it.